library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -81,8 +83,8 @@ class Client() {
     lateinit var contacts: Contacts
     lateinit var conversations: Conversations
     var logger: XMTPLogger = XMTPLogger()
-    var libXMTPClient: FfiXmtpClient? = null
     val libXMTPVersion: String = getVersionInfo()
+    private var libXMTPClient: FfiXmtpClient? = null
```

Comment:
I think this should be private. Clients shouldn't be able to access the V3 Rust stuff directly by passing any sdk level checks we have on them.